mysql-test/main/mysqld--help.result

```diff
@@ -719,9 +719,10 @@ The following specify which files/extra groups are read (specified before remain
  costs. disable_max_seek = Disable 'max_seek optimization'
  for secondary keys and slight adjustment of filter cost.
  disable_forced_index_in_group_by = Disable automatic
- forced index in GROUP BY. This variable will be deleted
- in MariaDB 11.0 as it is not needed with the new 11.0
- optimizer.
+ forced index in GROUP BY. fix_innodb_cardinality =
+ Disable doubling of the Cardinality for InnoDB secondary
+ keys. This variable will be deleted in MariaDB 11.0 as it
+ is not needed with the new 11.0 optimizer.
```

Comment:
The 11.0 branch is no longer maintained. Should the documentation only mention long-term-support branches, such as 11.4?

Reply:
I think the statement "is it is not needed with the new 11.0 optimizer" should stay there  even if 11.0 is EOL-ed.

Then, "This variable will be deleted in MariaDB 11.0 " can stay as well in order to not be confusing.  Changing 11.0 to 11.4 doesn't make the message much better.